test/test_hyperbolic_serre_green_naghdi_1d.jl

```diff
@@ -163,26 +163,26 @@ EXAMPLES_DIR = joinpath(examples_dir(), "hyperbolic_serre_green_naghdi_1d")
                                      "hyperbolic_serre_green_naghdi_dingemans.jl"),
                             tspan=(0.0, 1.0),
                             l2=[
-                                0.2462244051802164,
-                                0.8029763168465812,
-                                2.7915052205763788e-15,
-                                0.5551546220119555,
-                                0.24713044468457937,
+                                0.22618134328164993,
```

Comment:
These changes (and the ones further down for the SGN equations) are because the Dingemans initial condition was slightly differently defined for the SGN equations.